### DISCUSSION TITLE
init_io_gather

### DISCUSSION THREAD

**Discussion by dsidoren** [FESOM/fesom2#181] [General]
https://github.com/FESOM/fesom2/blob/98b11078e28f380c2602a8f57ca74d46ca5d0030/src/io_meandata.F90#L825

@hegish , could you remind why we call it twice (here three lines above)? 

thanks, Dima

**Comment by hegish** [comment]
that looks like an error to me. it should be called once only

**Reply by dsidoren** [reply]
could it be the reason for all our problems? :)

**Reply by hegish** [reply]
No, sorry. The second call does nothing. Like, really nothing. It is guarded with a logical.

**Reply by patrickscholz** [reply]
Which in the long list of our problems you are referring to ?
